### DISCUSSION TITLE
Where have the community plugins moved to?

### DISCUSSION THREAD

**Discussion by ZenSoftware** [dotansimha/graphql-code-generator#9726] [Q&A] [answered]
I am looking for the source for `@graphql-codegen/c-sharp` and `@graphql-codegen/c-sharp-operations`.  I saw in the [GraphQL Code Generator v5 Roadmap](https://github.com/dotansimha/graphql-code-generator/issues/9456) that the community plugins are planned to being moved.  I've searched around everywhere and I can't find the source!  Any help would be appreciated.  Thanks!

**Comment by saihaj** [comment, marked as answer]
https://github.com/dotansimha/graphql-code-generator-community/tree/main/packages/plugins/c-sharp
